lib/Debugger.ProtocolHandler/DebuggerImpl.h

```diff
@@ -25,73 +25,96 @@ namespace JsDebug
         DebuggerImpl& operator=(const DebuggerImpl&) = delete;
 
         // protocol::Debugger::Backend implementation
-        protocol::Response enable() override;
```

Comment:
I reordered the implemented methods alphabetically to make adding them easier.  The order is unchanged in the implementation file.